app/assets/javascripts/analytics-cross-domain-redirects.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env jquery */
```

Comment:
@andysellick - I think it makes sese to merge this here, but much of this code looks really similar to what we did in finder-frontend.

As x-domain is a shared problem would it make sense for this to migrate up into the components gem at some point?

Reply:
Yes, I think this should go straight into the components gem.

Reply:
You think it's ok to merge this here then refactor after or do you think this should get split off now?

Reply:
This is fine for now, I'd made a note on https://trello.com/c/p3UtkNtE/486-resolve-the-two-copies-of-analytics-code to migrate it as part of that work.

Reply:
I think this code is also duplicated in collections and finder-frontend, so I'll check that too.